dali/operators/reader/loader/coco_loader.h

```diff
@@ -39,26 +39,36 @@ class CocoLoader : public FileLabelLoader {
  public:
   explicit inline CocoLoader(
     const OpSpec& spec,
+    std::vector<int> &heights,
```

Comment:
This class is created in `dali/operators/reader/loader/loader_test.cc:134`, you need to adjust that test as now it won't compile.